ui/imports/shared/views/chat/MessageContextMenuView.qml

```diff
@@ -72,11 +72,13 @@ StatusPopupMenu {
     }
 
     function show(userNameParam, fromAuthorParam, identiconParam, textParam, nicknameParam, emojiReactionsModel) {
-        userName = userNameParam || ""
-        nickname = nicknameParam || ""
-        fromAuthor = fromAuthorParam || ""
-        identicon = identiconParam || ""
-        text = textParam || ""
+
+    // Not Refactored Yet
```

Comment:
@osmaczko feel free to remove lines 76-81, that is not needed any more, it was used before to set root props immediately before we open it. So that's not applicable any more.

Reply:
Sure, done